File: FechaTest.py
class Fecha:

    def __init__(self):

        self.mes = 0
        self.dia = 0
        self.anio = 0

    def mostrarFecha(self):

        print(
            self.mes,
            "/",
            self.dia,
            "/",
            self.anio
        )


fecha = Fecha()

fecha.mes = int(input("Ingresa el mes: "))

fecha.dia = int(input("Ingresa el día: "))

fecha.anio = int(input("Ingresa el año: "))


print("Fecha:")

fecha.mostrarFecha()

File: FechaTest.java
import java.util.Scanner;

public class FechaTest {

    public static void main(String args[]) {

        Fecha fecha = new Fecha();

        Scanner input = new Scanner(System.in);

        System.out.println("Ingresa el mes: ");
        fecha.mes = input.nextInt();

        System.out.println("Ingresa el día: ");
        fecha.dia = input.nextInt();

        System.out.println("Ingresa el año: ");
        fecha.anio = input.nextInt();

        System.out.println("Fecha: ");

        fecha.mostrarFecha();
    }
}